autoquran/src/main/java/com/quran/labs/autoquran/di/ServiceModule.kt

```diff
@@ -0,0 +1,29 @@
+package com.quran.labs.autoquran.di
+
+import com.quran.data.page.provider.QuranDataModule
+import com.quran.data.source.PageProvider
+import com.quran.data.source.QuranDataSource
+import com.quran.labs.autoquran.QuranAudioService
+import dagger.Component
+import dagger.Module
+import dagger.Provides
+
+@Module
+class ServiceModule {
```

Comment:
can we not use the existing module for this

Reply:
No as i believe we run it as different application.

Reply:
I believe due to different application of autoquran it require new graph of app and i don't wanna bload by adding everything.

Reply:
I already did this in my branch.